src/components/CollectionDetailPage/CollectionDetailPage.tsx

```diff
@@ -199,6 +199,31 @@ export default class CollectionDetailPage extends React.PureComponent<Props, Sta
     return null
   }
 
+  renderUnsyncedCollectionNoticeStatus() {
+    const { collection, status } = this.props
+
+    if (!collection?.isApproved || ![SyncStatus.UNSYNCED, SyncStatus.UNDER_REVIEW].some(s => s === status)) {
```

Comment:
```suggestion
    if (!collection?.isApproved || ![SyncStatus.UNSYNCED, SyncStatus.UNDER_REVIEW].incudes(status)) {
```
another possible and shorter way to do the same thing